sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/implementation/DeploymentExportResultImpl.java

```diff
@@ -17,11 +20,11 @@ final class DeploymentExportResultImpl extends
         implements
         DeploymentExportResult {
 
-    private ObjectMapper mapper;
+    private SerializerAdapter serializerAdapter;
 
     DeploymentExportResultImpl(DeploymentExportResultInner innerModel) {
         super(innerModel);
-        mapper = new ObjectMapper();
+        serializerAdapter = SerializerFactory.createDefaultManagementSerializerAdapter();
```

Comment:
Note that a simple ObjectMapper is not exactly same as SerializerAdapter created from azure-core. Latter has lots of configure which is for Azure JSON (e.g. date-time, duration, etc.).

ObjectMapper more like the innerMapper in https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/util/serializer/JacksonAdapter.java#L103

Maybe not a big deal here, as the ARM template should be simple JSON with string/numeric etc. It should not involve any advanced type like DateTime.

Reply:
We may need advice from Alan on whether `SerializerAdapter` is still the recommended class for serialization/de-serialization.

Reply:
Yeah. I'm seeing Alan's PR doing that:
https://github.com/Azure/azure-sdk-for-java/pull/40716/files#diff-3f1e47c051070c3f2d4a385d788c972fa02611ca0f9c9cf2652e40eefa45a683R30-R31

And the serializerAdapter is `JacksonAdapter`:
https://github.com/Azure/azure-sdk-for-java/pull/40716/files#diff-1d04b9c2cc5bc129be708cce645415ee0a8f8489515563b8ab21e0be132a96b0R102

@alzimmermsft Would you help confirm that we can substitute `ObjectMapper` for `JacksonAdapter`? And is this PR using it the correct way?